synapseutils/sync.py

```diff
@@ -1021,9 +1157,55 @@ def syncToSynapse(
         _manifest_upload(syn, df)
 
 
-def _manifest_upload(syn, df):
+def _convert_cell_in_manifest_to_python_types(
+    cell: str,
+) -> typing.Union[typing.List, datetime.datetime, float, int, bool, str]:
+    """
+    Takes a possibly semi-colon delimited cell from the manifest TSV file into a list
+    of items to be used as annotations.
+
+    Args:
+        cell: The cell item to convert.
+
+    Returns:
+        The list of items to be used as annotations. Or a single instance if that is
+            all that is present.
+    """
+    values_to_return = []
+
+    cell_values = re.split(pattern=COMMA_PATTERN, string=cell)
```

Comment:
Is it confusing that setting multiple annotations via a fileview is different from setting multiple annotations via the entity itself?

Reply:
The biggest difference is around escaping the comma in this case.

Edit annotation in file view:
![image](https://github.com/Sage-Bionetworks/synapsePythonClient/assets/17128019/983f28ce-80bb-4083-adb8-987c8291add4)

Edit annotation on the File:
![image](https://github.com/Sage-Bionetworks/synapsePythonClient/assets/17128019/84e11b3c-58a4-465c-ae9f-7a624934dd06)

Annotation on the Manifest TSV:
Which yes is different from setting it via the manifest TSV:
![image](https://github.com/Sage-Bionetworks/synapsePythonClient/assets/17128019/9057d0ab-2b4e-40a2-8fe5-c2cda6eac118)

Annotation in Python:
![image](https://github.com/Sage-Bionetworks/synapsePythonClient/assets/17128019/4f00cde8-3935-461b-b4df-7e3fe79e1bda)